Realm/RLMSyncUser.h

```diff
@@ -35,6 +35,15 @@ typedef NS_ENUM(NSUInteger, RLMSyncUserState) {
 /// A block type used for APIs which asynchronously vend an `RLMSyncUser`.
 typedef void(^RLMUserCompletionBlock)(RLMSyncUser * _Nullable, NSError * _Nullable);
 
+/// A block type used to report the progress of a permissions set operation.
```

Comment:
"A block type used to report the progress of a permission apply or revoke operation."?